GoogleDataTransport/GDTCORTests/Unit/GDTCORTransformerTest.m

```diff
@@ -125,6 +148,41 @@ - (void)testWriteEventWithTransformersThatCreateANewEvent {
                                       XCTAssertTrue(wasWritten);
                                       XCTAssertNil(error);
                                     }]);
+
+  [self waitForExpectations:bgTaskExpectations timeout:0.5];
+}
+
+#pragma mark - Helpers
+
+/** Sets  GDTCORApplicationFake handlers to expect the begin and the end of a background task with
+ * the specified name.
+ *  @return An array with the task begin and end XCTestExpectation.
+ */
+- (NSArray<XCTestExpectation *> *)expectationsBackgroundTaskBeginAndEndWithName:
+    (NSString *)expectedName {
+  XCTestExpectation *beginExpectation = [self expectationWithDescription:@"Background task begin"];
+  XCTestExpectation *endExpectation = [self expectationWithDescription:@"Background task end"];
+
+  GDTCORBackgroundIdentifier taskID = rand();
```

Comment:
I know it doesn't matter here, but I'd prefer something in the arc4random family just to prevent this code from being flagged for archaic API usage by external analyzers.